Resources/Prototypes/Entities/Objects/Devices/cartridges.yml

```diff
@@ -61,4 +61,23 @@
         state: server
     - type: NetProbeCartridge
 
-
+- type: entity
+  parent: BaseItem
+  id: LogProbeCartridge
+  name: LogProbe cartridge
+  description: A program for getting access logs from devices
+  components:
+    - type: Sprite
+      sprite: Objects/Devices/cartridge.rsi
+      state: cart-y
+    - type: Icon
+      sprite: Objects/Devices/cartridge.rsi
+      state: cart-y
```

Comment:
bumping Julian's review, ping a spriter (or the role) in the discord and see if anyone can sprite a cartridge variant for this, disregard if nobody steps up